build.sbt

```diff
@@ -348,6 +348,7 @@ lazy val `kamon-spring` = (project in file("instrumentation/kamon-spring"))
       // check that dependencies are shaded
       kanelaAgent % "provided",
       "org.springframework.boot" % "spring-boot-starter-web" % "2.4.2" % "provided",
```

Comment:
@SimunKaracic do you know down to what version are we backwards compatible?

Reply:
No idea, but I'll be decrementing and seeing where we fail

Reply:
We work up to `2.2.0` for sure! 
We probably work for older versions than that, but the examples start breaking when we go further back.